server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const reqString = {
+    type: String,
+    require: true
+};
+
+const ProfileSchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        require: true,
+        ref: 'user'
+    },
+    firstName: reqString,
+    lastName: reqString,
+    email: reqString,
+    description: {
+        type: String,
+        maxlength: 300
+    },
+    gender: String,
+    birthDate: Date,
+    phoneNumber: [String],
```

Comment:
I'm curious - any  reason the phone number is a string array instead of just string?

Reply:
May be the user has more than one phone number. In the document that I got from you in dropbox, the Edit Profile picture shows the system allows the user can put more than one phone number. I do based on that.

Reply:
I see! With that intent, your data type makes sense. :) 

Here's my copy of the Edit Profile mock. Usually, we only have one number associated to an account. Here, it appears that there's only an `Add a phone number button` since there's no associated phone number yet.
![setup-profile-edit](https://user-images.githubusercontent.com/78225194/135557096-48706329-dfe9-4256-8375-7baa2de961b5.png)
`